spec/features/sign_in_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+require 'rails_helper'
+
+feature 'sign in' do
+  scenario 'user signs in successfully' do
+    FactoryGirl.create(:user)
+    visit root_path
+    click_link "Log in"
+    fill_in "Email", with: "harry@potter.com"
+    fill_in "Password", with: "HarryPotter"
+    click_button "Log in"
```

Comment:
Nice.  Eventually I would recommend making a helper method to sign in a user, since this is likely something you will have to do a lot as part of test setups.